cmake/sanitize.cmake

```
@@ -34,5 +34,8 @@ if (NOT ${FLAG_DISPLAY_NAME})
     "please try another compiler or omit TBB_SANITIZE variable")
 endif()
 
+set(TBB_LSAN_OPTIONS "suppressions=${CMAKE_CURRENT_SOURCE_DIR}/cmake/lsan.suppressions")
+set(TBB_TESTS_ENVIRONMENT LSAN_OPTIONS=${TBB_LSAN_OPTIONS})
```

```suggestion
set(TBB_LSAN_ENVIRONMENT LSAN_OPTIONS=suppressions=${CMAKE_CURRENT_SOURCE_DIR}/cmake/lsan.suppressions)
```

`TBB_LSAN_OPTIONS` is used only in one place, so maybe just merge two options into one.

---

it's good point merge two variable in one for now, but I would like to use neutral name for this variable.
```suggestion
set(TBB_TESTS_ENVIRONMENT ${TBB_TESTS_ENVIRONMENT} LSAN_OPTIONS="suppressions=${CMAKE_CURRENT_SOURCE_DIR}/cmake/lsan.suppressions")
```